solutions/cf-863B.mdx

```diff
@@ -2,19 +2,19 @@
 id: cf-863B
 source: CF
 title: Kayaking
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Brad Ma
 ---
 
 [Official Editorial](https://codeforces.com/blog/entry/54708)
 
 **Time Complexity:** $\mathcal{O}(N^3)$
 
+## Implementation
```

Comment:
above time complexity

Reply:
the time complexity is there i'm not sure what you mean

Reply:
he meant to put this above time complexity